libkbfs/folder_block_ops.go

```diff
@@ -2588,6 +2586,7 @@ func (fbo *folderBlockOps) getDeferredWriteCountForTest(lState *lockState) int {
 }
 
 func (fbo *folderBlockOps) updatePointer(kmd KeyMetadata, oldPtr BlockPointer, newPtr BlockPointer) {
+	fbo.log.CDebugf(context.Background(), "Updating reference for pointer %s to %s", oldPtr.ID, newPtr.ID)
```

Comment:
I'm going to leave it for now because of debugging, but I'll add a `TODO` to remove it.